modules/Makefile

```diff
@@ -86,7 +86,20 @@ MODULES_TO_DOCUMENT = \
 	standard/UtilReplicatedVar.chpl \
 	$(SYS_CTYPES_MODULE_DOC)
 
+DISTS_TO_DOCUMENT = \
```

Comment:
Actually, can you just alphabetize DIST_TO_DOCUMENT, I'll take care of INTERNAL_MODULES_TO_DOCUMENT in an upcoming PR